src/doc/src/reference/manifest.md

```diff
@@ -151,7 +151,7 @@ documentation = "https://docs.rs/bitflags"
 
 > **Note**: [crates.io] may not show certain sites if they are known to not be
 > hosting documentation and are possibly of malicious intent e.g., ad tracking
-> networks. At this time, the site `rust-ci.org` is not allowed.
+> networks. At this time; `rust-ci.org`, and `rustless.org` are not allowed.
```

Comment:
I would prefer that we don't list the individual sites that are blocked in the documentation. And in general, I don't think the **Note** is actually necessary.  Registries like crates.io will have various mitigations for abuse in all sorts of different scenarios, and I don't think we need to call them all out.  Perhaps the entire Note should be removed?

Reply:
Seems reasonable to me, I also thought it was a bit odd to call them out, I've updated the PR to remove the note instead.

Reply:
FWIW, looks like PR title and content diverged here. I was quite confused at first when reading the title and the diff. ;)